protos/guard/guard.proto

```diff
@@ -118,14 +128,17 @@ message FileStoreMeta {
     (gogoproto.nullable) = false,
     (gogoproto.stdtime) = true
   ]; //rent end time
-  int32 check_frequency = 6; // shard check frequency in a year,  1 means check once per year, 0 means check 12 per year(1 per month)
+  int32 check_frequency = 6; //reserve: shard check frequency in days, 1 means check once per day
   int64 guard_fee = 7; // reserve: the fee which guard will charge for supporting
   int64 escrow_fee = 8; //reserve: the fee which escrow will charge for supporting
   int32 shard_count = 9; //shard's count, by default is 30
   int32 minimum_shards = 10; //minimum count of shards which can recover the whole file, by default is 10
   int32 recover_threshold = 11; //reserved field, indicate the threshold for starting to recover
   string escrow_pid = 12;
   string guard_pid = 13;
+  int32 warn_challenge_times_limit = 14; //reserve: indicate how many times warn challenge will take the contract to cancel status, renter cannot set it now, use 7 as default
+  int32 success_challenge_times_limit = 15; //reserve: indicate how many times success will take the contract from warn status back to right, renter cannot set it now, use 1 as default
+  int32 check_frequency_warn = 16; //reserve: check frequency in days while in warn status, renter cannot set it now, use 1 as default
```

Comment:
is this days or times?

Reply:
it uses times in one year before, but now I found check_in_days is more understandable, but I already use term check_frequency in the original data model before, to avoid impact the current data, I keep to use the term now. Anyway it is reserved field, so I just change the original comment now.